### PR TITLE
chore: lock node version in circleci

### PR DESCRIPTION
**Pull request by stephenrob**
Locks down the node version in CircleCI to `18.17.1` this was defaulting to `20.11.0`

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -22,8 +22,7 @@ commands:
 
   node_setup:
     steps:
-      - node/install:
-          node-version: "" # use .nvmrc
+      - node/install
       - node/install-packages:
           include-branch-in-cache-key: false
```
Comment (mrliptontea) [inline .circleci/config.yml:27]: Actually it looks like we can remove `node_setup` completely.

Comment (stephenrob) [reply .circleci/config.yml:27]: It's still referenced by the end to end tests and commitlint steps

Comment (mrliptontea) [reply .circleci/config.yml:27]: Oh, ok, didn't see it in the diff so assumed it was only needed by printnodeversion.

**Review by mrliptontea — APPROVED** [review]

**Pull request merged**